packages/realm/src/OrderedCollection.ts

```diff
@@ -433,33 +536,57 @@ export abstract class OrderedCollection<T = unknown, EntryType extends [unknown,
+   * @params predicate.value - The value of the element.
+   * @params predicate.index - The index of the element.
+   * @params predicate.obj - The object being traversed.
+   * @params thisArg - An object to which the this keyword can refer in the predicate function. If thisArg is omitted, undefined is used as the this value.
+   * @returns The value of the first element in the array that satisfies the provided testing function. Otherwise, undefined is returned.
    */
   find<S extends T>(
     predicate: (this: void, value: T, index: number, obj: T[]) => value is S,
     thisArg?: any,
   ): S | undefined;
   /**
    * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find Array.prototype.find}
+   * @params predicate - A function that accepts up to three arguments. The find method calls the predicate function one time for each element in the collection.
+   * @params predicate.value - The value of the element.
+   * @params predicate.index - The index of the element.
+   * @params predicate.obj - The object being traversed.
+   * @params thisArg - An object to which the this keyword can refer in the predicate function. If thisArg is omitted, undefined is used as the this value.
+   * @returns The value of the first element in the array that satisfies the provided testing function. Otherwise, undefined is returned.
    */
   find<T>(predicate: (value: T, index: number, obj: T[]) => unknown, thisArg?: any): T | undefined;
   find(predicate: (value: T, index: number, obj: T[]) => boolean, thisArg?: any): T | undefined {
     return [...this].find(predicate, thisArg);
   }
   /**
    * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex Array.prototype.findIndex}
+   * @params predicate - A function that accepts up to three arguments. The findIndex method calls the predicate function one time for each element in the collection.
+   * @params thisArg - An object to which the this keyword can refer in the predicate function. If thisArg is omitted, undefined is used as the this value.
+   * @returns The index of the first element in the array that satisfies the provided testing function. Otherwise, -1 is returned.
    */
   findIndex(predicate: (value: T, index: number, obj: readonly T[]) => unknown, thisArg?: any): number {
     return [...this].findIndex(predicate, thisArg);
   }
   // TODO: Implement support for RealmObjects, by comparing their #objectKey values
   /**
    * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes Array.prototype.includes}
+   * @params searchElement - The element to search for.
+   * @params fromIndex - The position in this array at which to begin searching for searchElement. A negative value searches from the index of array.length + fromIndex by asc.
+   * @returns true if the searchElement is found in the array; otherwise, false.
```

Comment:
If we don't support that, than we should definitely make a note for it.  It isn't obvious to me what doesn't work, but i'll look into it.

Reply:
I've added a note, but I left the description there.  If we do implement this, then we should also support negative indexing.

Reply:
It's from [this code](https://github.com/realm/realm-js/blob/783a329649c3a7bf27c7e19ac1805b6a0c86c129/packages/realm/src/OrderedCollection.ts#L334-L340), and also Core is not using it and we're not passing it to Core. Yeah a note would be great 🙂

Reply:
Btw, it also says `by asc` but not sure what this would imply..? I don't think we would change how the search is done if using a negative index, merely from where the searching starts.